lib/buildsystems/autotools/v1.rb

```diff
@@ -0,0 +1,43 @@
+require 'package'
+
+class Autotools_v1 < Package
+  @_buildsystems_autotools_version = '1.0.0'
+
+  def self.configure_options (options = '')
+    if options
+      @configure_options = options
+    end
+    return @configure_options
+  end
+
+  def self.check? (bool = false)
+    @check = bool
+    return @check
+  end
+
+  def self.build
+    # Run autoreconf if necessary
+    unless File.executable? './configure'
+      if File.executable? './autogen.sh'
+        system 'NOCONFIGURE=1 ./autogen.sh'
+      elsif File.executable? './bootstrap'
+        system 'NOCONFIGURE=1 ./bootstrap'
```

Comment:
Same here.

Reply:
`--no-configure` is a parameter and should be placed after `./autogen.sh`

Reply:
Good point, I'm embarrassed I missed that. I've updated both of the suggestions.

Reply:
The line could be:
```
system 'NOCONFIGURE=1 ./bootstrap'
```
or:
```
system './bootstrap --no-configure'
```
but it doesn't need to be both.

Reply:
Let's use NOCONFIGURE=1, because the script may react with an error if passed an incorrect parameter, but environment variables are ignored if not used.

Reply:
But, from what I've checked, not all scripts honor both. So it makes more sense to put both in just in case.

Reply:
I think that makes sense, its not beautiful but it's in the backend so it dosent really matter.

Reply:
see [e5ce5a1](https://github.com/chromebrew/chromebrew/pull/6306/commits/e5ce5a12cfa4f6a0720bf3040087a1855b00baa4)